src/test/java/teammates/test/cases/action/InstructorCourseAddActionTest.java

```diff
@@ -124,7 +127,10 @@ public void testExecuteAndPostProcess() throws Exception {
                               Const.ParamsNames.COURSE_TIME_ZONE, "UTC");
         redirectResult = getRedirectResult(addAction);
 
-        String expectedDestination = Const.ActionURIs.INSTRUCTOR_COURSES_PAGE + "?error=false&user=idOfInstructor1OfCourse1";
+        String expectedDestination = getPageResultDestination(
+                                             Const.ActionURIs.INSTRUCTOR_COURSES_PAGE,
+                                             false,
+                                             "idOfInstructor1OfCourse1");
```

Comment:
Looks okay except that 8 spaces should be used instead of 9 in the snippet provided; in addition you may want to put the parameters in a single line as well.